metastore/if/hive_metastore.thrift

```diff
@@ -1550,4 +1562,4 @@ const string TABLE_TRANSACTIONAL_PROPERTIES = "transactional_properties",
 
 // Keys for alter table environment context parameters
 const string EXPECTED_PARAMETER_KEY = "expected_parameter_key",
-const string EXPECTED_PARAMETER_VALUE = "expected_parameter_value",
\ No newline at end of file
+const string EXPECTED_PARAMETER_VALUE = "expected_parameter_value",
```

Comment:
Minor: is this change needed?

Reply:
Not really - this is done automatically by my editor.

Reply:
I think it's an improvement to remove trailing whitespaces and empty line at the end so left these changes there.

Reply:
I'm OK with either style. It's funny that google java formatter requires an empty line at every file.